councilmatic_core/migrations/0041_event_extras.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Generated by Django 1.9.13 on 2018-05-07 16:20
+from __future__ import unicode_literals
+
+import django.contrib.postgres.fields.jsonb
+from django.db import migrations
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('councilmatic_core', '0040_mediaevent_meta'),
+    ]
+
+    operations = [
+        migrations.AddField(
+            model_name='event',
+            name='extras',
+            field=django.contrib.postgres.fields.jsonb.JSONField(default=dict),
+        ),
+        migrations.RunSQL('''
+            UPDATE councilmatic_core_event
+              SET extras = json_build_object('guid', guid)::jsonb
```

Comment:
I think there's a `jsonb_build_object` too that would make it so you don't need to cast it.

Reply:
it's [not available in postgres 9.4](https://www.postgresql.org/docs/9.4/static/functions-json.html) (which is what we're running in travis; unsure if it's safe to bump)

> Table 9-42 shows the functions that are available for creating json values. (Currently, there are no equivalent functions for jsonb, but you can cast the result of one of these functions to jsonb.)